src/components/filter/expert/expert-filter-constants.ts

```diff
@@ -674,6 +695,42 @@ export const FIELDS_OPTIONS = {
         dataType: DataType.NUMBER,
         inputType: 'number',
     },
+    CONVERTER_STATION_ID_1: {
+        name: FieldType.CONVERTER_STATION_ID_1,
+        label: 'converterStationId1',
+        dataType: DataType.STRING,
+    },
+    CONVERTER_STATION_ID_2: {
+        name: FieldType.CONVERTER_STATION_ID_2,
+        label: 'converterStationId2',
+        dataType: DataType.STRING,
+    },
+    CONVERTERS_MODE: {
+        name: FieldType.CONVERTERS_MODE,
+        label: 'convertersMode',
+        dataType: DataType.ENUM,
```

Comment:
Even this field is declared as enum but translation is not done

![Capture d’écran du 2024-07-19 15-13-06](https://github.com/user-attachments/assets/d2d0fda6-6773-4786-a47a-61902724dffe)

I have a suggestion in in PR: https://github.com/gridsuite/commons-ui/pull/530